test/unit/services/s3.test.js

```diff
@@ -80,3 +86,138 @@ describe('listObjects', () => {
     ]);
   });
 });
+
+// TODO: test listRawFileNames. this function calls listObjects which makes a call to aws api
+// might need to refactor listRawFileNames to allow for mocking a return value for listObjects
+
+describe('getObject', () => {
+  test('Returns a buffer', async () => {
+
+  // set up mocks
```

Comment:
Spacing seems a tad off. Can you try formatting it (right click -> `Format Document`).

Our linter doesn't run against the `test/` folder so that's why it doesn't complain.